lib/services/app.js

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+
+import errorHandler, { notFound } from '../middleware/error_handler';
+import apiv1 from '../routes/api.v1';
+
+const app = express();
+
+app.use('/api/v1', apiv1);
+app.use('/', notFound);
+app.use(errorHandler);
```

Comment:
I'm prefer to use separate routes, for example:
routes.map(require).forEach(function (route) {
  route(app);
});
what do you think?

Reply:
@akosenko I'm not sure that I understand you. Would you might writing a example code with that preferences?

Reply:
well, lets imagine that you have two routes - event.js and account.js in routes directory. To initialize them we'll write smth like that
app.js
------------------------------------------------------
const routes = globSync('./app/routes/**/*.js', {
  cwd: __dirname
}).map(require);
routes.forEach(function (route) {
  route(app);
});

event.js
------------------------------------------------------
module.exports = function (app) {
  const accountRouter = express.Router();
  accountRouter.get
  accountRoute.post
  ....
  app.use('api/v1', accountRouter);
}

Reply:
this approach will provide modularity and easy way to write tests for each route

Reply:
It's a nice idea to move routes with the same part of route name to own files